server/bytestream/bytestream.go

```diff
@@ -0,0 +1,84 @@
+package bytestream
+
+import (
+	"context"
+	"flag"
+	"io"
+	"net/url"
+	"strings"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+
+	"google.golang.org/grpc"
+
+	bspb "google.golang.org/genproto/googleapis/bytestream"
+)
+
+func StreamBytestreamFile(ctx context.Context, uri string, callback func([]byte)) error {
```

Comment:
This is basically https://github.com/buildbuddy-io/buildbuddy/blob/master/server/util/grpc_client/grpc_client.go + https://github.com/buildbuddy-io/buildbuddy-internal/blob/master/enterprise/server/remote_execution/cachetools/cachetools.go#L23. Would b e nice to consolidate sometime.

Reply:
Nice - will try to consolidate in a follow up